backend-tests/tests/test_auditlogs.py

```diff
@@ -425,7 +435,7 @@ def _test_args_sort(self, tenant_users, events):
                 check_log(resp[i], case["expected"][i])
 
 
-def make_deployment(token: str) -> Tuple[Dict, Dict]:
+def make_deployment(token: str, device: str = "arm1") -> Tuple[Dict, Dict]:
```

Comment:
Please consider renaming this argument to `device_id` - why its default value is `arm1`? It looks more like device type, not device ID.